cli/cmd/common.go

```diff
@@ -22,3 +23,15 @@ func handleCmdErr(cmd *cobra.Command, err error) {
 		log.Fatalf(err.Error())
 	}
 }
+
+// errNoConfig is returned if environment config file tt.yaml not found.
+var errNoConfig = errors.New("tt.yaml not found, you need to create tt" +
```

Comment:
Let's use ConfigName from the `configure` package instead of `tt.yaml`.

Reply:
Thank you, updated.